docs/source/reference/envvars.rst

```diff
@@ -161,6 +161,16 @@ These variables influence what is printed out during compilation of
 
    If set to non-zero, print out debugging information about type inference.
 
+
+.. envvar:: NUMBA_SYS_MONITORING_DEFAULT
```

Comment:
```suggestion
.. envvar:: NUMBA_ENABLE_SYS_MONITORING
```
should it be named this to be consistent with other on/off type of env vars, cf. profiling? Default in which case would be `off` == 0.

Reply:
Renamed in https://github.com/numba/numba/pull/9780/commits/31c0af021d491fae47c2d276e5db2dab2617a8f4

Reply:
Thanks.